pkg/argparser/args.go

```diff
@@ -83,7 +83,7 @@ func analyzeArgs(am *argMap, args string) {
 	}
 }
 
-func GetArgs(args string, state *state.HelmState) []string {
+func GetArgs(args string, state *state.HelmState, withDiffArgs bool) []string {
```

Comment:
@ yktakaha4 We can make this func more flexible. there will be more args like applyArgs, sync Args. WDYT?

Reply:
@yxxhero Thank you for your review.
Should I modify it so that I can get the type of subcommand currently being executed from `HelmState` , or define a struct like `GetArgsOptions` ?
I would be happy to hear your opinions on how to make it more maintainable.

Reply:
@yktakaha4  we can try to define a struct like GetArgsOptions.

Reply:
Fixed to use GetArgsOptions.

In the current usage of GetArgs, It seems possible to change the interface to `GetArgs(c ApplyConfigProvider, state *state.HelmState, opts *GetArgsOptions)` and decide whether to include diffArgs inside GetArgs.
Should I make this change?
https://github.com/helmfile/helmfile/blob/a08b9860e34446d31e1f92adb706d6bb913bf00a/pkg/app/app.go#L1372-L1375